Sources/Apodini/Semantic Model Builder/OpenAPI/Document/OpenAPIDocumentBuilder.swift

```diff
@@ -13,7 +13,7 @@ struct OpenAPIDocumentBuilder {
     var pathsObjectBuilder: OpenAPIPathsObjectBuilder
     var componentsObjectBuilder: OpenAPIComponentsObjectBuilder
 
-    init(configuration: OpenAPIConfiguration) {
+    init(configuration: inout OpenAPIConfiguration) {
```

Comment:
As `OpenAPIConfiguration` is a class using inout doesn't have an effect here as it is a reference type and any mutations will be done on a single instance.

Reply:
👍 09f5109 - leftover, `OpenAPIConfiguration` once was a struct